SimCalorimetry/EcalSimAlgos/src/EcalShapeBase.cc

```diff
@@ -5,7 +5,6 @@
 #include <algorithm>
 
 #include <iostream>
-using namespace std;
```

Comment:
this is OK in .cc files

Reply:
even if OK, since it was not needed, I decided to remove it also from the .cc file